animal_crossing.py

```diff
@@ -131,8 +121,8 @@ def _stalk_list_sale_prices(cmd):
 
 	output = []
 	for result in results.values():
-		expires_in = _calculate_expiration(dateutil.parser.parse(price['expiration']) - current_time)
-		price_str = ('%s: %d (Expires in %dh %ds)' %
+		expires_in = _calculate_expiration(dateutil.parser.parse(result['expiration']) - current_time)
+		price_str = ('%s: %d (Expires in %dh %dm)' %
```

Comment:
I dunno if it's better or worse, but https://github.com/raylu/sbot/search?q=readable_rel&unscoped_q=readable_rel is already used in 2 places so you could reuse it here